README.md

```
@@ -1,151 +1,95 @@
-- [Gobuster](https://github.com/OJ/gobuster)
-- [GitLeaks](https://github.com/zricethezav/gitleaks) & [GitDumper](https://github.com/internetwache/GitTools/tree/master/Dumper)
-- [Log4j Scan](https://github.com/fullhunt/log4j-scan)
-- [Spring4Shell Scan](https://github.com/fullhunt/spring4shell-scan)
-- [CMSeeK](https://github.com/Tuhinshubhra/CMSeeK/)
-- [OWASP JoomScan](https://github.com/OWASP/joomscan)
-- [OWASP ZAP](https://www.zaproxy.org/)
-- [Nikto](https://github.com/sullo/nikto)
-- [Nuclei](https://github.com/projectdiscovery/nuclei)
-- [SearchSploit](https://www.exploit-db.com/searchsploit)
-- [Metasploit](https://www.metasploit.com/)
+Join our community in [![](https://readmecodegen.vercel.app/api/social-icon?name=discord&size=10) Discord](https://discord.gg/Zyduu5C7M3) and follow us on [![](https://readmecodegen.vercel.app/api/social-icon?name=x&size=10)](https://x.com/rekonosec).
```

markdownlint-cli2 (0.23.2)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)